src/packages/migrate/package.json

```diff
@@ -36,11 +37,12 @@
     "jest": "26.2.2",
     "lint-staged": "10.2.11",
     "make-dir": "3.1.0",
-    "pkg-up": "3.1.0",
     "prettier": "2.0.5",
     "react": "16.11.0",
+    "tempy": "0.5.0",
     "ts-jest": "26.1.4",
-    "typescript": "3.9.7"
+    "typescript": "3.9.7",
+    "ws": "7.3.1"
```

Comment:
I tried to not add dependencies as much as possible, but I needed this to be able to communicate with Studio server, since a vanilla Node implementation would be extremely verbose.

I also thought about exposing the Websocket endpoint in studio-server as a REST endpoint, but that turned out to be much more complicated too.